# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ooo/t2-distro.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./patches/src680/apply.orig	2005-03-24 21:38:35.816671912 -0400
+++ ./patches/src680/apply	2005-03-24 21:39:16.540480952 -0400
@@ -32,6 +32,8 @@
 PLD64: PLDBase, 64bit
 # Win32
 Win32 : Win32Common, NovellOnly, NovellOnlyWin32
+# www.t2-project.org
+T2-Project : LinuxCommon
 
 # -------- [ Tag ], patch sets --------
 
